### PR TITLE
Create sample-claude-multimodal.json

### PR DESCRIPTION
**Pull request by mollyexten**

### CODE REVIEW

File: gen-ai/sample-claude-multimodal.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          "type": "noauth"
+        },
+        "url": {
+          "host": [
+            "{{api.url}}"
+          ],
+          "raw": "{{api.url}}"
+        }
+      },
+      "id": "84734c07-4338-4591-a494-805339a396bb",
```
Comment (dabelche) [inline gen-ai/sample-claude-multimodal.json:74]: What is this ID?

File: gen-ai/sample-claude-multimodal.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
+  "variable": [
+    {
+      "value": "https://api.anthropic.com/v1/messages",
+      "key": "api.url"
+    },
+    {
+      "value": "claude-3-opus-20240229",
```
Comment (dabelche) [inline gen-ai/sample-claude-multimodal.json:8]: I really wish there was a way to include comments here with haiku and sonnet...

File: gen-ai/sample-claude-multimodal.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      "current": "",
+      "key": "imgBase64"
+    },
+    {
+      "value": "0",
+      "current": "",
+      "key": "temp"
+    },
+    {
+      "value": "<your-api-key>",
```
Comment (dabelche) [inline gen-ai/sample-claude-multimodal.json:27]: is it worth specifying that this is your anthropic API key so it's not confused with mabl API key?

File: gen-ai/sample-claude-multimodal.json [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    },
+    {
+      "value": "<image-in-base64>",
+      "current": "",
+      "key": "imgBase64"
+    },
+    {
+      "value": "0",
+      "current": "",
+      "key": "temp"
```
Comment (dabelche) [inline gen-ai/sample-claude-multimodal.json:24]: you're sure temp is working here?  For me it was throwing some error converting what looked like a string to a number...

File: gen-ai/sample-claude-multimodal.json (hunk 3 above)
Comment (mollyexten) [reply gen-ai/sample-claude-multimodal.json:27]: Good point. Maybe `<anthropic-api-key>`?

File: gen-ai/sample-claude-multimodal.json (hunk 4 above)
Comment (mollyexten) [reply gen-ai/sample-claude-multimodal.json:24]: It works for me. The API Test Editor highlights it as an error even though it isn't: https://mabl.atlassian.net/issues/MABL-13595

File: gen-ai/sample-claude-multimodal.json (hunk 1 above)
Comment (mollyexten) [reply gen-ai/sample-claude-multimodal.json:74]: Postman request ID - will remove

File: gen-ai/sample-claude-multimodal.json (hunk 2 above)
Comment (mollyexten) [reply gen-ai/sample-claude-multimodal.json:8]: What if we added a console log statement? Would that be too subtle??

**Review by mollyexten — COMMENTED** [review]
@dabelche I'll follow up on these changes tomorrow. I only have commenting permissions.

**Pull request closed without merging**